engines/wintermute/ad/ad_item.cpp

```diff
@@ -581,6 +600,25 @@ bool AdItem::scCallMethod(ScScript *script, ScStack *stack, ScStack *thisStack,
 		return STATUS_OK;
 	}
 
+#ifdef ENABLE_FOXTAIL
+	//////////////////////////////////////////////////////////////////////////
+	// [FoxTail] RemoveHoverCursor
+	// Used while changing cursor type at some included script
+	// Return value is never used
+	//////////////////////////////////////////////////////////////////////////
+	else if (strcmp(name, "RemoveHoverCursor") == 0) {
+		stack->correctParams(0);
+
+		if (_cursorHover) {
```

Comment:
Unneccesary check, delete of nullptr is a safe no-op, so you can reduce this to just "delete _cursorHover;"

Reply:
Thanks! Fixed at 6060826512a0c9021cabc4cea0bc2d0879eef688